datasets/gbif/requirements.txt

```
@@ -0,0 +1,6 @@
+git+https://github.com/TomAugspurger/gbif
+git+https://github.com/TomAugspurger/stac-table
```

```suggestion
git+https://github.com/stac-utils/stac-table@99a30be419baa2445ab6d0756629eea079c80972
```

And I can work on a release of this at some point. There is a 1.0.0 at https://pypi.org/project/stac-table/, but I don't know exactly what source code that refers to.